src/github/nag.rs

```diff
@@ -1219,11 +1219,13 @@ impl<'a> RfcBotComment<'a> {
     }
 
     fn add_comment_url(issue: &Issue, msg: &mut String, comment_id: i32) {
-        let url = format!("https://github.com/{}/{}/{}#issuecomment-{}",
-                             issue.repository,
-                             issue.number,
-                             if issue.is_pull_request { "pull" } else { "issues" },
-                             comment_id);
+        let url = format!(
+            "https://github.com/{repo}/{typ}/{number}#issuecomment-{id}",
+            repo=issue.repository,
```

Comment:
Can you put spaces between `=` for readability?

Reply:
Done.
Could I do a rustfmt PR later?

Reply:
Sure.